### PR TITLE
Update help site URLs

### PR DESCRIPTION
**Pull request by DanHillesheim**
remove /article/  and help.bitwarden.com references

## Type of change

- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

Update links to correct location

## Code changes

link updates

## Testing requirements

Test links for accuracy

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Comment by vincentsalucci** [comment]
This looks fine to me! @DanHillesheim Is there a backing task on Asana so QA can check all these links once this is merged?

**Comment by Hinton** [comment]
Do we plan on updating the other clients as well? I know some of the links are used across our products.

**Comment by DanHillesheim** [comment]
@vincentsalucci  No, there isn't a task in Asana. Would this go in the "Issues (Engineering)" board? I don't see an obvious board. 

@Hinton Fred and I would love for those links to be updated as well. The general pattern is:

`help.bitwarden.com /` -> `bitwarden.com/help/`
`bitwarden.com/help/article/*` -> `bitwarden.com/help/*`

**Comment by Hinton** [comment]
@DanHillesheim "Planned Dev"

File: src/app/settings/change-password.component.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export class ChangePasswordComponent extends BaseChangePasswordComponent {
         );
         if (learnMore) {
           this.platformUtilsService.launchUri(
-            "https://help.bitwarden.com/article/attachments/#fixing-old-attachments"
+            "https://bitwarden.com/help/attachments/#fixing-old-attachments"
```
Comment (Hinton) [inline src/app/settings/change-password.component.ts:90]: This link seems broken?

File: src/app/vault/vault.component.html [hunk 2]
```diff
@@ -128,7 +128,7 @@ <h1>
           {{ "providers" | i18n }}
           <a
             class="ml-auto"
-            href="https://bitwarden.com/help/article/about-providers/"
+            href="https://bitwarden.com/help/about-providers/"
```
Comment (Hinton) [inline src/app/vault/vault.component.html:131]: Link is broken

**Review by Hinton — COMMENTED** [review]
Two broken links. Looks good otherwise

File: src/app/vault/vault.component.html (hunk 2 above)
Comment (DanHillesheim) [reply src/app/vault/vault.component.html]: fixed

**Review by Hinton — APPROVED** [review]

**Pull request merged**